apollo-compiler/src/main/kotlin/com/apollostack/compiler/TypeSpecBuilder.kt

```diff
@@ -0,0 +1,8 @@
+package com.apollostack.compiler
+
+import com.squareup.javapoet.TypeSpec
+
+interface TypeSpecBuilder {
```

Comment:
Also, this would force them to implemented a `packageName` method, which isn't really relevant to their functionality.